05.ls/ls.rb

```diff
@@ -2,42 +2,49 @@
 
 # frozen_string_literal: true
 
-NUM_OF_COLUMNS = 3
+COLUMN_COUNT = 3
 
-def calc_num_of_row(num_of_objects)
-  (num_of_objects / NUM_OF_COLUMNS.to_f).ceil
+def main
+  dir_items = Dir.entries('.').sort
+  removed_hidden_dir_items = remove_hidden_dir_items(dir_items)
```

Comment:
少し名前が長いのでこんなふうにしてみると良いかも。

```suggestion
  filtered_dir_items = filter_hidden_dir_items(dir_items)
```

Reply:
修正しました！